students/LuisGonzalez/session07/html_render.py

```diff
@@ -0,0 +1,112 @@
+
+class Element(object):
+    tag = "html"
+    indent = 1
+    def __init__(self, content = None, **kwargs):
+
+        self.content = content
+
+        if len(kwargs) > 0:
+            self.extraAttributes = str(kwargs).strip('{}')
+            self.extraAttributes = self.extraAttributes.replace("'", "", 2)
+            self.extraAttributes = " " + self.extraAttributes.replace("'", '"')
+        else:
+            self.extraAttributes = None
+        if self.content is None:
+            self.content = list()
+        else:
+            self.content = [content]
+    def append(self, new_content):
+        self.content.append(new_content)
+
+    def render(self, file_out, ind=""):
+        if self.tag == "html":
+            file_out.write("<!DOCTYPE html>\n")
+
```

Comment:
Did you test out your flow control here with each possible type of tag and scenario?